engines/director/debugger.cpp

```diff
@@ -329,6 +333,60 @@ bool Debugger::cmdBacktrace(int argc, const char **argv) {
 bool Debugger::cmdDisasm(int argc, const char **argv) {
 	Lingo *lingo = g_director->getLingo();
 	if (argc == 2) {
+		if (Common::String(argv[1]).equals("all")) {
```

Comment:
Python detected :) It is more readable, but I'd write `if (!strcmp(argv[1], "all"))`

Reply:
Fixed